src/build123d/build_common.py

```diff
@@ -776,33 +776,19 @@ def __init__(self, *objs: Union[Face, Plane, Location]):
 
 #
 # To avoid import loops, Vector add & sub are monkey-patched
-def _vector_add(self: Vector, vec: VectorLike) -> Vector:
-    """Mathematical addition function where tuples are localized if workplane exists"""
-    if isinstance(vec, Vector):
-        result = Vector(self.wrapped.Added(vec.wrapped))
-    elif isinstance(vec, tuple) and WorkplaneList._get_context():
-        result = Vector(self.wrapped.Added(WorkplaneList.localize(vec).wrapped))  # type: ignore[union-attr]
-    elif isinstance(vec, tuple):
-        result = Vector(self.wrapped.Added(Vector(vec).wrapped))
-    else:
-        raise ValueError("Only Vectors or tuples can be added to Vectors")
-
-    return result
-
-
-def _vector_sub(self: Vector, vec: VectorLike) -> Vector:
-    """Mathematical subtraction function where tuples are localized if workplane exists"""
-    if isinstance(vec, Vector):
-        result = Vector(self.wrapped.Subtracted(vec.wrapped))
-    elif isinstance(vec, tuple) and WorkplaneList._get_context():
-        result = Vector(self.wrapped.Subtracted(WorkplaneList.localize(vec).wrapped))  # type: ignore[union-attr]
-    elif isinstance(vec, tuple):
-        result = Vector(self.wrapped.Subtracted(Vector(vec).wrapped))
-    else:
-        raise ValueError("Only Vectors or tuples can be subtracted from Vectors")
-
-    return result
-
-
-Vector.add = _vector_add  # type: ignore[assignment]
-Vector.sub = _vector_sub  # type: ignore[assignment]
+
+def _vector_add_sub_wrapper(original_op: Callable[[Vector, VectorLike], Vector]):
+    def wrapped(self: Vector, vec: VectorLike):
```

Comment:
Could a different name be used here as `wrapped` is the name of a Vector property?  Otherwise, looks good.

Reply:
I don't think the name matters, does it show up anywhere? Either way I can change it, any suggestion?

Reply:
How about `wrapper`?